src/modules/ItemsCount.test.js

```diff
@@ -0,0 +1,44 @@
+/** @jest-environment jsdom */
+/* eslint-disable global-require */
```

Comment:
- [ ] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉

Reply:
We've removed the eslint disables. Thank you.

Reply:
we have fixed the linter disable error. thanks for the feedback!